src/database/models/rollouts.js

```diff
@@ -0,0 +1,117 @@
+const common = require('@constants/common')
+
+module.exports = (sequelize, DataTypes) => {
+	const Rollouts = sequelize.define(
+		'Rollouts',
+		{
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: DataTypes.INTEGER,
+			},
+			resource_type: {
+				allowNull: false,
+				type: DataTypes.STRING,
+			},
+			resource_id: {
+				type: DataTypes.INTEGER,
+			},
+			status: {
+				allowNull: false,
+				type: DataTypes.ENUM('PENDING', 'ROLLED_OUT', 'INACTIVE'),
+				defaultValue: 'PENDING',
+			},
+			rollout_date: {
+				allowNull: true,
+				type: DataTypes.STRING,
+			},
+			organization_id: {
+				primaryKey: true,
+				allowNull: false,
+				type: DataTypes.STRING,
+			},
+			user_id: {
+				allowNull: false,
+				type: DataTypes.STRING,
+			},
+			start_date: {
+				allowNull: true,
+				type: DataTypes.STRING,
+			},
+			end_date: {
+				allowNull: true,
+				type: DataTypes.STRING,
+			},
+			published_id: {
+				type: DataTypes.STRING,
+			},
+			title: {
+				allowNull: false,
+				type: DataTypes.STRING,
+			},
+			blob_path: {
+				allowNull: true,
+				type: DataTypes.STRING,
+			},
+			parent_id: {
+				type: DataTypes.INTEGER,
+			},
+			stage: {
+				allowNull: true,
+				type: DataTypes.ENUM('PROGRAM', 'SOLUTION'),
+			},
+			duplicate_template_id: {
+				type: DataTypes.STRING,
+			},
+			created_by: {
+				allowNull: false,
+				type: DataTypes.STRING,
+			},
+			updated_by: {
+				type: DataTypes.STRING,
+			},
+			created_at: {
```

Comment:
@adithyadinesh0412  created at deleted at and updatedat no need to mention in model